src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/OrderItem.xml

```diff
@@ -25,7 +31,12 @@
         </itemOperations>
 
         <property name="id" identifier="true" writable="false" />
-        <property name="adjustments" writable="false">
+        <property name="order" readable="true" />
+        <property name="productName" readable="true" />
+        <property name="product" readable="true">
+            <subresource resourceClass="%sylius.model.product.class%" />
```

Comment:
I'm not sure if the product should be a subresource here. Could you add a screenshot of an API response?

Reply:
Indeed there is no need, I will delete those lines with next changes.